packages/@okta/vuepress-theme-prose/util/geoLocation.js

```diff
@@ -0,0 +1,85 @@
+import storage from './localStorage'
+
+// Add/Remove country codes from this array to have them blocked/unblocked
+const BLOCKED_REGION_LIST = ['RU', 'BY'];
+const STORAGE_KEY = 'oktaLocation';
+const ONE_WEEK_IN_MILLISECONDS = 1000 * 60 * 60 * 24 * 7;
+
+class GeoLocation {
+  constructor(onCompletion) {
+    this.hasRetried = false;
+
+    if (typeof onCompletion === 'function') {
+      this.onCompletion = onCompletion;
+    }
```

Comment:
nit: could write this like
```js
this.onCompletion = typeof onCompletion === 'function' ? onCompletion : () => {};
```
then you don't need to guard the calls later on

Reply:
or you can use `?.` as a guard

Reply:
Even better: `constructor(onCompletion = () => {})`

Reply:
Nothing enforces that `onCompletion` is a function though